pkg/ingester/ingester.go

```diff
@@ -885,6 +888,17 @@ func (i *Ingester) Query(req *logproto.QueryRequest, queryServer logproto.Querie
 		}
 	}
 
+	defer func() {
+		status := "200"
+		if err != nil {
+			status = "400"
+		}
+		statsCtx := stats.FromContext(ctx)
+		execTime := time.Since(start)
+		logql.RecordIngesterSeriesQueryMetrics(ctx, i.logger, req.Start, req.End, req.Selector, status,
```

Comment:
I don't think this is a series request; this is a streams request.
`QuerySample`, which should also have logging I believe (?), returns series.

Reply:
Good idea, I've renamed it and also added logging to QuerySample.